src/Cryptogram.java

```diff
@@ -0,0 +1,37 @@
+import java.io.*;
+import java.util.*;
```

Comment:
You should never use `*` in imports.  Just import everything individually, your IDE should do this automatically.